JenkinsfileRT

```diff
@@ -43,7 +43,7 @@ jobconfig.publish_env_on_success_only = true
 jobconfig.publish_env_filter = "spacetelescope/master"
 
 // Define python version for conda
-python_version = "3.11"
+python_version = "3.9"
```

Comment:
A real fix for the regtest server issues has been applied to jwst/master. So please remove this change from your PR and rebase your PR branch to pull in those changes from master. You should be able to run a regtest on the PR branch then.